pkg/api/server/register_images.go

```diff
@@ -10,32 +10,652 @@ import (
+	//   '500':
+	//     description: unexpected error
+	//     schema:
+	//      "$ref": "#/types/ErrorModel"
+	r.Handle(VersionedPath("/images/{name:..*}/history"), APIHandler(s.Context, handlers.HistoryImage)).Methods(http.MethodGet)
+	// swagger:operation GET /images/{nameOrID}/json images inspectImage
+	//
+	// Inspect an image
+	//
+	// ---
+	// parameters:
+	//  - in: path
+	//    name: nameOrID
+	//    required: true
+	//    description: the name or ID of the container
+	// produces:
+	// - application/json
+	// responses:
+	//   '200':
+	//     schema:
+	//     items:
+	//       "$ref": "#/types/imageInspect"
+	//   '404':
+	//     description: no such image
+	//     schema:
+	//       "$ref": "#/types/ErrorModel"
+	//   '500':
+	//     description: unexpected error
+	//     schema:
+	//      "$ref": "#/types/ErrorModel"
 	r.Handle(VersionedPath("/images/{name:..*}/json"), APIHandler(s.Context, generic.GetImage))
+	// swagger:operation POST /images/{nameOrID}/tag images tagImage
+	//
+	// Tag an image
+	//
+	// ---
+	// parameters:
+	//  - in: path
+	//    name: nameOrID
+	//    required: true
+	//    description: the name or ID of the container
+	//  - in: query
+	//    name: repo
+	//    type: string
+	//    description: the repository to tag in
+	//  - in: query
+	//    name: tag
+	//    type: string
+	//    description: the name of the new tag
+	// produces:
+	// - application/json
+	// responses:
+	//   '201':
+	//     description: no error
+	//   '400':
+	//     description: bad parameter
+	//     schema:
+	//       "$ref": "#/types/ErrorModel"
+	//   '404':
+	//     description: no such image
+	//     schema:
+	//       "$ref": "#/types/ErrorModel"
+	//   '409':
+	//     description: conflict
+	//     schema:
+	//       "$ref": "#/types/ErrorModel"
+	//   '500':
+	//     description: unexpected error
+	//     schema:
+	//      "$ref": "#/types/ErrorModel"
 	r.Handle(VersionedPath("/images/{name:..*}/tag"), APIHandler(s.Context, handlers.TagImage)).Methods(http.MethodPost)
-
-	// commit has a different endpoint
+	// swagger:operation POST /commit/ commit commitContainer
+	//
+	// Create a new image from a container
+	//
+	// ---
+	// parameters:
+	//  - in: query
+	//    name: container
+	//    type: string
+	//    description: the name or ID of a container
+	//  - in: query
+	//    name: repo
+	//    type: string
+	//    description: the repository name for the created image
+	//  - in: query
+	//    name: tag
+	//    type: string
+	//    description: tag name for the created image
+	//  - in: query
+	//    name: comment
+	//    type: string
+	//    description: commit message
+	//  - in: query
+	//    name: author
+	//    type: string
+	//    description: author of the image
+	//  - in: query
+	//    name: pause
+	//    type: bool
+	//    description: pause the container before committing it
+	//  - in: query
+	//    name: changes
+	//    type: string
+	//    description: instructions to apply while committing in Dockerfile format
```

Comment:
Needs a requestBody stanza

Reply:
we dont actually support that, so i need to document that.